### ISSUE TITLE
No italic in Viz.js map?

### ISSUE THREAD

**Issue by tonysepia** [christianvoigt/argdown#138]
Example:

<Question>: Will the text surrounded by *asterisks* be in italic in Viz.JS Map?

^ the above works correctly in Html view, but not in Viz.JS Map

VS Code 1.43.0
Argdown 1.3.0

**Comment by christianvoigt** [comment]
Hi, sorry for the late answer. You are right, this is a missing feature and should be fixed in the next release.

**Comment by antonostrovsky** [comment]
Thank you very much!

**Comment by christianvoigt** [comment]
I tried to fix this in 1.4.0 but it ended up to be a little bit of a hack: Graphviz currently "swallows" any empty spaces behind bold or italic ranges. To get a visually acceptable result, I cheated by automatically inserting extra empty spaces behind the ranges. This might cause problems if you export to GraphML and edit the map in yEd.

Links are not supported by Graphviz at all (only tables and cells can have href attributes), so I had to leave them out completely. :(

Let's hope that Graphviz fixes this bug so that I can remove this hack again.

**Comment by antonostrovsky** [comment]
Such a busy commit! https://github.com/christianvoigt/argdown/commit/4896628ef6a7255a00364cadc90e91580f7a47e3 This was a lot of work to get these features going. Thank you for this!

**Comment by christianvoigt** [comment]
Yes, it is somewhat complicated because the line breaks have to be inserted before adding markup, which makes applying the text ranges a mess.

I think the bold ranges are still not displayed correctly, even with the additional space. I really should open an issue at the Graphviz repository.

I am in the process of releasing the VSCode extension  (a Webpack nightmare) and updating the docs. @argdown/cli 1.4.5 has already been released. The rest should be done in a few hours.

**Comment by christianvoigt** [comment]
The issue is caused by [this issue](https://github.com/aduh95/viz.js/issues/10) in the Viz.js library.

Help is welcome: it would be fantastic if anyone could help out by compiling Graphviz and finding out which libraries are missing in Viz.js (see the link to the issue for further details).

This issue is also causing #214.
